src/core_modules/capture-core/components/WidgetNote/NoteSection/NoteSection.tsx

```diff
@@ -120,14 +113,22 @@ const NoteSectionPlain = ({
         </div>
     );
 
-
     return (
         <div className={classes.wrapper}>
             <div className={classes.notesWrapper}>
                 {notes
                     .sort((a, b) => moment(a.storedAt).valueOf() - moment(b.storedAt).valueOf())
-                    .map(note => <NoteItem key={`note-item-${note.note}-`} {...note} />)
-                }
+                    .map((note) => {
+                        const noteValue = note.note || note.value;
+                        return (
+                            <NoteItem
+                                key={`note-item-${noteValue}-`}
+                                value={noteValue}
+                                storedAt={note.storedAt}
+                                createdBy={note.createdBy}
+                            />
+                        );
+                    })}
```

Comment:
Use this exact line instead: '.map(note => <NoteItem key={`note-item-${note.note || note.value}-`} {...note} />)'